product_plm_import/models/product_plm.py

```diff
@@ -84,7 +85,25 @@ def _get_uom(self):
             uom = self.env.ref("uom.product_uom_unit")
         return uom
 
-    def _create_product(self):
+    def _get_quality_check_category(self):
+        if self.acceptance_test_categ:
+            quality_check_categ = self.env["quality.check.category"].search(
+                [("code", "=", self.acceptance_test_categ)], limit=1
+            )
+            if quality_check_categ:
+                return quality_check_categ
+            self.write(
+                {
+                    "error_message": _(
+                        "Quality Check Category (%s) does not exist.",
+                        self.acceptance_test_categ,
+                    ),
+                    "state": "failed",
+                }
+            )
+        return False
```

Comment:
To make it slightly more concise, adding the missing `self.ensure_one()`.
```suggestion
        self.ensure_one()
        quality_check_categ = self.env["quality.check.category"].browse()
        test_categ = self.acceptance_test_categ
        if test_categ:
            quality_check_categ = self.env["quality.check.category"].search(
                [("code", "=", test_categ)], limit=1
            )
            if not quality_check_categ:
                self.error_message = _(
                    "Quality Check Category (%s) does not exist.", test_categ,
                )
                self.state = "failed"
        return quality_check_categ
```